.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,86 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+
+jobs:
+  e2e-tests:
+    if: ${{ github.ref_name == 'main' || github.actor == github.repository_owner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        PORTAL:
+          [
+            adknowledgeportal,
+            arkportal,
+            bsmn,
+            cancercomplexity,
+            challengeportal,
+            digitalhealth,
+            elportal,
+            nf,
+            psychencode,
+            stopadportal,
+          ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/pnpm-setup-action
+      - name: Build portal
+        run: |
+          cd "${{ env.PORTALS_SUBDIR }}"
+          chmod +x ./build.sh
+          ./build.sh "${{ matrix.PORTAL }}"
+      - name: Start portal
+        run: pnpm nx run portals:portal:start
+      - name: Install Playwright Browsers
+        run: |
+          cd "${{ env.PORTALS_SUBDIR }}"
+          pnpm playwright install --with-deps
+      # Name report in playwright.config once feature is released:
+      # https://github.com/microsoft/playwright/issues/27284
+      - name: Run e2e tests
+        run: PWTEST_BLOB_REPORT_NAME="${{ matrix.PORTAL }}" pnpm nx run portals:e2e
```

Comment:
Name the Playwright report with the Portal name, so the HTML report includes tags for each Portal -- 

<img width="1098" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/c493f95b-492b-4ac0-89a6-bdf890a90a7d">

Reply:
Love the tags!